src/api/index.js

```diff
@@ -4,11 +4,13 @@ import placements from './placements'
 import sections from './sections'
 import grades from './grades'
 import students from './students'
+import similarStudent from './similar-student'
 
 export default ({ config, db }) => {
 	let api = Router()
 
 	api.use('/placements', placements({ config, db }))
+	api.use('/similar-student', similarStudent({ config, db }))
```

Comment:
I think it'd be better to stick with /similar-student. We ran into some trouble with resource-router-middleware and having multiple levels of endpoints, and I figured our time was better spent on other problems